rules/linux/execution_nice_binary.toml

```
@@ -24,9 +24,9 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by host.id, process.pid with maxspan=1m
-[process where event.type == "start" and process.name == "nice" and process.args : ("/bin/bash", "/bin/sh", "/bin/dash", "sh", "bash", "dash")]
-[process where process.name : ("bash", "sh", "dash")]
+sequence with maxspan=30s
+[process where event.type == "start" and process.name == "nice" and process.args in ("/bin/bash", "/bin/sh", "/bin/dash", "sh", "bash", "dash")] by process.entity_id
+[process where process.name in ("bash", "sh", "dash")] by process.parent.entity_id
```

woo! Happy to see `process.entity_id` back, but we can also flatten this one

```suggestion
process where event.type == "start" and process.name in ("bash", "sh", "dash") and
  process.parent.name == "nice" and process.parent.args in ("/bin/bash", "/bin/sh", "/bin/dash", "sh", "bash", "dash")
```